src/components/cylc/Tree.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <!-- mx is margin on the x axis, and mb is margin-bottom, 3 is 3 out of 5 -->
+  <div class="mx-3 mb-3">
+    <!-- each workflow is a tree root -->
+    <tree-item
+        v-for="workflow of workflows"
+        :key="workflow.id"
+        :node="workflow"
+        :hoverable="hoverable"
+        :min-depth="minDepth"
+        :expanded="expanded"
+        v-on:tree-item-created="onTreeItemCreated"
+        v-on:tree-item-expanded="onTreeItemExpanded"
+        v-on:tree-item-collapsed="onTreeItemCollapsed"
+        v-on:tree-item-clicked="onTreeItemClicked"
+    >
+    </tree-item>
+  </div>
+</template>
+
+<script>
+import TreeItem from '@/components/cylc/TreeItem'
+
+export default {
+  name: 'Tree',
+  props: {
+    workflows: {
+      type: Array,
+      required: true
+    },
```

Comment:
You may want to add one empty line between multi-line properties https://vuejs.org/v2/style-guide/#Empty-lines-in-component-instance-options-recommended

Reply:
Hadn't seen that part about styling in Vue docs, thanks. Do you think we need that here? It looks fine for me, but I use PyCharm on a large screen. It seems like they say either way is fine, but spaces may help devs with other editors such as Vim (paging @hjoliver as I think he uses vim?).

Reply:
I use vim and/or VS Code; no strong opinion on this at this stage!

Reply:
>no strong opinion on this at this stage!

Same here. We may have to adjust the other components to follow this pattern too. But perhaps we can do it later? Unless it is difficult to other to read the code, then happy to update this PR with an extra commit for styling.

Reply:
> I use vim and/or VS Code; no strong opinion on this at this stage!

It's fine, a personal style choice really ;-)